docs/scos/dev/guidelines/monitorable-process-guidelines.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Monitorable process guidelines
+description: @TODO
+last_updated: @TODO
+template: concept-topic-template
+related:
+- title: NFR guidelines
+  link: docs/scos/dev/guidelines/nfr-guidelines.html
+- title: Operatable feature guidelines
+  link: docs/scos/dev/guidelines/operatable-feature-guidelines.html
+- title: Process documentation guidelines
+  link: docs/scos/dev/guidelines/process-documentation-guidelines.html
+---
+
+In order to enable Operations Team to track/correlate issues in the operated/deployed applications, the following elements need to be applied over
+* log generation
+* and metric generation.
+
+## Log generation
+Applies to all application components.
+
+### Log entry best practices
+* Log entry format and text taxonomy MUST be unified across all log providers for easy human processing.
```

Comment:
do we have a standard for the log entry format in core that we can suggest to project level functionality?